docs/tutorials/tutorial_run_tests_locally.md

```diff
@@ -0,0 +1,257 @@
+    aws-gardener_prod_trustedboot-amd64-tofu-test-platform                        Run platform tests build with OpenTofu for aws-gardener_prod_trustedboot-amd64
+    aws-gardener_prod_trustedboot-arm64-tofu-test-platform                        Run platform tests build with OpenTofu for aws-gardener_prod_trustedboot-arm64
+    azure-gardener_prod-amd64-tofu-test-platform                                  Run platform tests build with OpenTofu for azure-gardener_prod-amd64
+    azure-gardener_prod-arm64-tofu-test-platform                                  Run platform tests build with OpenTofu for azure-gardener_prod-arm64
+    azure-gardener_prod_tpm2_trustedboot-amd64-tofu-test-platform                 Run platform tests build with OpenTofu for azure-gardener_prod_tpm2_trustedboot-amd64
+    azure-gardener_prod_tpm2_trustedboot-arm64-tofu-test-platform                 Run platform tests build with OpenTofu for azure-gardener_prod_tpm2_trustedboot-arm64
+    azure-gardener_prod_trustedboot-amd64-tofu-test-platform                      Run platform tests build with OpenTofu for azure-gardener_prod_trustedboot-amd64
+    azure-gardener_prod_trustedboot-arm64-tofu-test-platform                      Run platform tests build with OpenTofu for azure-gardener_prod_trustedboot-arm64
+    gcp-gardener_prod-amd64-tofu-test-platform                                    Run platform tests build with OpenTofu for gcp-gardener_prod-amd64
+    gcp-gardener_prod-arm64-tofu-test-platform                                    Run platform tests build with OpenTofu for gcp-gardener_prod-arm64
+    gcp-gardener_prod_tpm2_trustedboot-amd64-tofu-test-platform                   Run platform tests build with OpenTofu for gcp-gardener_prod_tpm2_trustedboot-amd64
+    gcp-gardener_prod_tpm2_trustedboot-arm64-tofu-test-platform                   Run platform tests build with OpenTofu for gcp-gardener_prod_tpm2_trustedboot-arm64
+    gcp-gardener_prod_trustedboot-amd64-tofu-test-platform                        Run platform tests build with OpenTofu for gcp-gardener_prod_trustedboot-amd64
+    gcp-gardener_prod_trustedboot-arm64-tofu-test-platform                        Run platform tests build with OpenTofu for gcp-gardener_prod_trustedboot-arm64
+  </pre>
+</details>
+
+
+Great. You see a lot of potential test targets now. You know the name of the image you want to test (you can continue with `kvm-gardener_prod`), and you know the architecture of the image you want to test (you can continue with your local CPU arch, as long as it is arm64 or amd64).
+
+We have the full name of the Garden Linux image, for example `kvm-gardener_prod-amd64`
+
+But the make targets list a lot more than just the flavours, which one to pick? 
+This is easy once you understand the different methods to run the Garden Linux VM we want to test. 
+
+Since we want to run tests locally, we focus here on the two local options:
```

Comment:
maybe here it would make sense to quickly elaborate on what the alternative to running test 'locally' means.. this might not be obvious to the reader